### ISSUE TITLE
There is excess cutting of cyrillic characters

### ISSUE THREAD

**Issue by grabus** [gregjacobs/Autolinker.js#111]
```javascript
Autolinker.link('https://ru.wikipedia.org/wiki/Табурет')
```
I expect this:
```html
<a href="https://ru.wikipedia.org/wiki/" target="_blank">ru.wikipedia.org/wiki/Табурет</a>
```
but I get this:
```html
<a href="https://ru.wikipedia.org/wiki/" target="_blank">ru.wikipedia.org/wiki</a>Табурет
```

**Comment by Katochimoto** [comment]
:+1:

**Comment by gregjacobs** [comment]
Fixed in 0.24.0
